src/main/scala/org/broadinstitute/dsde/workbench/avram/api/CollectionsServlet.scala

```diff
@@ -1,54 +1,44 @@
 package org.broadinstitute.dsde.workbench.avram.api
 
 import java.util.UUID
+import javax.servlet.http.HttpServletResponse
 
 import io.circe.generic.auto._
-import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
+import javax.ws.rs._
+import javax.ws.rs.core.Response.Status
+import javax.ws.rs.core.{HttpHeaders, Response}
+
 import org.broadinstitute.dsde.workbench.avram.model.{AvramException, SamResource}
 import org.broadinstitute.dsde.workbench.avram.service.CollectionsService
 import org.broadinstitute.dsde.workbench.avram.util.AvramResult
+import org.broadinstitute.dsde.workbench.avram.Avram
 
 import scala.util.Try
 
-
-class CollectionsServlet extends HttpServlet with AvramServlet {
+@Path("/api/collections/v1")
+class CollectionsServlet(avram: Avram) extends AvramServlet(avram) {
```

Comment:
This isn't technically a servlet anymore. Maybe rename these back to `*Endpoint`. Jersey/JAX-RS is a framework similar to Cloud Endpoints Framework.